labs/learning-tokens.md

```diff
@@ -43,8 +43,27 @@ https://github.com/Jmarvin-12
 
 https://github.com/AlfonsoGovela
 
+https://github.com/TanjinAlam
```

Comment:
If you would like to add a new committer to the lab, please file a ticket under [hyperledger-labs/governance issues](https://github.com/hyperledger-labs/governance/issues/new/choose) using the "Add someone to the Hyperledger Labs GitHub org" issue type.

Reply:
@tkuhrt Yes, I wanted to update [wiki](https://labs.hyperledger.org/labs/learning-tokens.html) page of learning token. We have recenetly made our publication and we made reference of this.

Reply:
Thank you @tkuhrt for your help and support. How do we get the second reviewer? Should I ask Vipin, or Kamlesh? Apologies for pushing. We are in a hurry because yesterday GBBC published our Special Report Learning Tokens, and we would like to have the readme section of our repo aligned to this document.